lib/src/revset.rs

```diff
@@ -103,6 +103,8 @@ pub enum RevsetParseErrorKind {
     },
     #[error("Invalid arguments to revset function \"{name}\": {message}")]
     InvalidFunctionArguments { name: String, message: String },
+    #[error("Revset function \"mine\" requires a user name")]
```

Comment:
nit: "requires a configured user name", perhaps, so it doesn't sound like you're supposed to pass a name as function argument

Reply:
Now using a `&str` based on what `UserSettings::user_email()` returns instead of an `Option<&str>`, so I removed the check.